route.js

```diff
@@ -17,15 +17,63 @@ app.listen(port, (err) => {
 
 
 app.get('/', (request, response) => {
-    response.send('Bienvenue sur Express');
+  response.send('Bienvenue sur Express');
 });
   
 app.get('/users', (req, res) => {
-    connection.query('SELECT * from DVM_Legal_Entity', (err, results) =>{
-        if(err) {
-            res.status(500).send('No body was here..')
-        } else {
-            res.json(results);
-        }
-    })
-});
\ No newline at end of file
+  connection.query('SELECT * from DVM_Legal_Entity', (err, results) =>{
+    if(err) {
+      res.status(500).send('No body was here..')
+    } else {
+      res.json(results);
+    }
+  })
+});
+
+app.get('/users/:id', (req, res) => {
+  const { id } = req.params
+  connection.query('SELECT * from DVM_Legal_Entity WHERE id = ?', id, (err, results) => {
+    if(err) {
+      res.status(500).send('Nobody was here..')
+    } else {
+      res.json(results);
+    }
+  })
+})
+
+app.post('/users', (req, res) => {
+  const formData = req.body;
+  connection.query('INSERT INTO DVM_Legal_Entity SET ?', formData, (err, results) => {
+    if (err) {
+      console.log(err);
+      res.status(500).send("Erreur lors de la sauvegarde d'un DVM");
+    } else {
+      res.sendStatus(200);
```

Comment:
creation is a 201, and according to rest specification you should send back the created resource